libraries/vault_config.rb

```diff
@@ -63,6 +64,9 @@ class VaultConfig < Chef::Resource
       attribute(:api_addr, kind_of: String)
       attribute(:cluster_addr, kind_of: String)
       attribute(:disable_clustering, equal_to: [true, false])
+      # Seal options
+      attribute(:seal_type, kind_of: String)
+      attribute(:seal_options, option_collector: true, default:{})
```

Comment:
Should be "default: {}".  Other cookstyle alerts are generated and should be reviewed and fixed.

Reply:
The cookstyle alerts should be fixed.  I don't know what's going on with circleci.  I downloaded and converged and tested the cookbook. It passes existing tests.  Tests of adding the ui and seal options would be a good addition.

This change looks good otherwise.